### PR TITLE
changes

### PR DESCRIPTION
**Pull request by kpslp**
made changes, the link for tenant docs wont work as i've not committed the docs as pr is not approve(as of the time im doing this)

### CODE REVIEW

File: docs/whats-new/staging-release-notes.md [hunk 1]
```diff
@@ -2,6 +2,35 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 29 March 2023
+
+Frontend version: 1.0.0-20230322.1044 | Backend version: 1.55.6-230323.1031
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md]: the backend version is 1.55.7-230328.0536

File: docs/whats-new/staging-release-notes.md [hunk 2]
```diff
@@ -2,6 +2,35 @@
...
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Tenants can retry or stop a webhook trigger on TechPass portal</summary>
+
+Tenants can now retry or stop a webhook trigger on TechPass portal. This option is useful when a code change required. Previously, the webhook was automatically re-triggered and expired after 14 days.
+
+Refer to [Retry or stop webhook trigger](https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/webhooks?id=retry-or-stop-webhook) for more information.
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md]: the link should be https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/webhooks?id=retry-or-stop-webhook-trigger

File: docs/whats-new/staging-release-notes.md (hunk 1 above)
Comment (kpslp) [reply docs/whats-new/staging-release-notes.md]: updated in new commit @JLeeChan

File: docs/whats-new/staging-release-notes.md (hunk 2 above)
Comment (kpslp) [reply docs/whats-new/staging-release-notes.md]: updated in new commit

File: docs/whats-new/staging-release-notes.md [hunk 3]
```diff
@@ -2,6 +2,35 @@
...
+## Staging release 29 March 2023
+
+Frontend version: 1.0.0-20230322.1044 | Backend version: 1.55.6-230323.1031
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Tenants can retry or stop a webhook trigger on TechPass portal</summary>
+
+Tenants can now retry or stop a webhook trigger on TechPass portal. This option is useful when a code change required. Previously, the webhook was automatically re-triggered and expired after 14 days.
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md:14]: I looked at the documentation in doc portal and I see the **How** to retry/stop the webhook.
- I think there is missing information on the possible sending states of the webhook event (i.e. failed, retrying, stopped, and <more i missed?>)
- can we elaborate on **Why** we need to retry or stop sending the webhook events? and their effects

File: docs/whats-new/staging-release-notes.md [hunk 4]
```diff
@@ -2,6 +2,35 @@
...
+
+Refer to [Retry or stop webhook trigger](https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/webhooks?id=retry-or-stop-webhook) for more information.
+</details>
+
+<details>
+<summary style="font-size:20px;font-weight:bold"> <i>user-deleted</i> webhook event now includes assigned groups</summary>
+
+The webhook event *user-deleted* includes all groups the user is assigned to that follow *namespace:group name*. This limits the groups to those belonging to a tenant and does not expose any TechPass internal groups the user is assigned to.
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md]: Can we change it to:
The webhook event *user-deleted* includes all groups the user is assigned to that follow the *namespace:group name* naming convention. 

I don't think we need to announce about the internal groups.

Comment (kpslp) [reply docs/whats-new/staging-release-notes.md]: updated

**Review by JLeeChan — APPROVED** [review]

**Pull request merged**